graviti/manager/dataset.py

```diff
@@ -26,6 +26,43 @@
 LazyLists = NestedDict[str, LazyList[Any]]
 
 
+class DatasetAccessInfo(ReprMixin, Mapping[str, str]):
```

Comment:
shall we put `ReprMixin` after `Mapping` since `Mapping` is a real base class and `ReprMixin` is a mixin class